lib/src/Pos/Client/CLI/Options.hs

```diff
@@ -50,6 +50,10 @@ commonArgsParser = do
     configurationOptions <- configurationOptionsParser
     pure CommonArgs{..}
 
+-- To get rid of abiguity.
```

Comment:
Would be nice to have just a little more context there about which ambiguity we're getting rid of.